src/transaction/txParser.ts

```diff
@@ -183,15 +238,20 @@ const parseTxCerts = (txCertificates: any[]): _Certificate[] => {
   )
 }
 
-const parseTxWithdrawals = (
-  withdrawals: any,
-): _Withdrawal[] => {
+const parseTxWithdrawals = (withdrawals: any): _Withdrawal[] => {
   if (!isWithdrawalsMap(withdrawals)) {
     throw Error(Errors.WithrawalsParseError)
   }
   return Array.from(withdrawals).map(([address, coins]): _Withdrawal => ({ address, coins: BigInt(coins) }))
 }
 
+const parseValidityIntervalStart = (validityIntervalStart: any): BigInt => {
+  if (isBigNumber(validityIntervalStart)) {
+    return BigInt(validityIntervalStart)
+  }
+  throw Error(Errors.ValidityIntervalStartParseError)
+}
+
 const parseUnsignedTx = ([txBody, meta]: _UnsignedTxDecoded): _UnsignedTxParsed => {
```

Comment:
Related to the previous commnet, somewhere here I d just add a condition with comment that mind fields are not supported for now.
```
if txBody.get(TxBodyKeys.MINT) throw Error(Minting not supported yet)
```
something like this.

Reply:
Fixed.